packages/realm/src/Realm.ts

```diff
@@ -381,7 +381,8 @@ export class Realm {
         return Realm.normalizePath(config.path);
       } else {
         const bindingSyncConfig = toBindingSyncConfig(config.sync);
-        return config.sync.user.internal.syncManager.pathForRealm(bindingSyncConfig, config.path);
+        const path = config.sync.user.internal.pathForRealm(bindingSyncConfig, undefined);
+        return path;
```

Comment:
Seems fine to keep as a simple return without a local `path` variable.

```suggestion
        return config.sync.user.internal.pathForRealm(bindingSyncConfig, undefined);
```

Reply:
Me debugging at some point